### PR TITLE
[Testcase]Add test case for tensor decoder image labelling

### PR DESCRIPTION
**Pull request by jinhyuck-park**
Add test case for tensor decoder for image labelling

Signed-off-by: jinhyuck-park <jinhyuck83.park@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Comment by ohsewon** [comment]
refer : https://github.com/nnsuite/testcases/tree/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant

**Comment by jinhyuck-park** [comment]
@myungjoo I cannot find getTestModels.sh in nnstreamer/tests/common or nnstreamer/common
do you mean make a 'new' getTestModels.sh file?

**Comment by myungjoo** [comment]
> @myungjoo I cannot find getTestModels.sh in nnstreamer/tests/common or nnstreamer/common
do you mean make a 'new' getTestModels.sh file?

Then, you need to pull from the upstream and rebase your commit.

**Comment by jinhyuck-park** [comment]
@myungjoo I see, this file newly updated so I updated with this file too

File: common/getTestModels.sh [hunk 1]
```diff
@@ -8,6 +8,6 @@ fi
 
 mkdir -p ${TARGET}
 pushd ${TARGET}
-	ln -s ${DIR}/../tests/nnstreamer_filter_tensorflow_lite/models/mobilenet_v1_1.0_224_quant.tflite .
-	ln -s ${DIR}/../tests/nnstreamer_filter_tensorflow_lite/labels/labels.txt .
+	ln -s ${DIR}/../tests/test_models/nnstreamer_filter_tensorflow_lite/models/mobilenet_v1_1.0_224_quant.tflite .
+	ln -s ${DIR}/../tests/test_models/nnstreamer_filter_tensorflow_lite/labels/labels.txt .
```
Comment (myungjoo) [inline common/getTestModels.sh]: Remove ```nnstreamer_filter_tensorflow_lite/``` from the two strings.

Comment (jinhyuck-park) [reply common/getTestModels.sh]: I fixed file path properly

**Pull request merged**